.github/known-licenses.txt

```diff
@@ -0,0 +1,22 @@
+Apache-1.0
+Apache-2.0
+BSD-2-Clause
+BSD-3-Clause
+curl
+GPL-1.0-or-later
+GPL-2.0-only
+GPL-3.0-only
```

Comment:
This would be quite worrying, but it looks like it is only conf-autoconf, only used during build and not at runtime.
In general having both GPL-3.0-only and GPL-2.0-only would be a problem if we ever linked them into the same binary...